### PR TITLE
docs: standardize code blocks

### PR DESCRIPTION
**Pull request by trueNAHO**

### CODE REVIEW

File: docs/reference.md [hunk 1]
```diff
@@ -59,8 +59,8 @@ generate disk images:
 2. **Execute the result file:** Execute the generated result file. Running
    `./result --help` will output the available options:
 
-   ```
-   ./result --help
+   ```bash
+   $ ./result --help
```
Comment (Mic92) [inline docs/reference.md:63]:
```suggestion
   ./result --help
```

**Comment by phaer** [comment]
nit: For interactive shell use "```console" or "```shellsession" (synonymous) often provide better highlighting then  "```bash".

**Comment by trueNAHO** [comment]
> nit: For interactive shell use "`console" or "`shellsession" (synonymous) often provide better highlighting then "```bash".

Should we convert all `bash` tags to `console`?

**Comment by phaer** [comment]
I don't care too much personally, but you could just drop the second commit?

**Comment by trueNAHO** [comment]
> you could just drop the second commit

I assume you are referring to https://github.com/nix-community/disko/commit/9abffdda130e9e3f27892979e9324baee4c17735.

In that case, https://github.com/nix-community/disko/pull/479/commits/6bfc47f3093d5dc19817a016d0fac0c2590241ab should do what you meant.

**Comment by Mic92** [comment]
@mergify queue

**Pull request merged**